core/storage/exploration/gae_models.py

```diff
@@ -201,6 +201,22 @@ class ExplorationRightsModel(base_models.VersionedModel):
     )
 
     def save(self, committer_id, commit_message, commit_cmds):
+        """Saves a new version of the exploration, updating the Exploration
+        datastore model.
+
+        Args:
+            committer_id: str. The user_id of the user who committed the
+                change.
+            commit_message: str. The commit description message.
+            commit_cmds: list(dict). A list of commands, describing changes
+                made in this model, which should give sufficient information to
+                reconstruct the commit. Each dict always contains:
+                    cmd: str. Unique command.
```

Comment:
> Unique command here means the command for that particular commit. I've included one example also. But I can't find the list where they are actually mentioned. Even by looking at the super classes, I can't find where exactly they are defined. Can you help me? Thanks!

They are in exp_domain.py. I think it's fine to say something like "cmd: str. The type of the command. A full list of command types can be found in core/domain/exp_domain.py."

Reply:
Ok, thanks! I've made the changes.